### ISSUE TITLE
version number related test failures in Scala community build

### ISSUE THREAD

**Issue by SethTisue** [FasterXML/jackson-module-scala#319]
sometime after abacab34f065abda49c4575a8dfb941498c02b3b (perhaps in 7c5fe0c6c632e353340fe02e227b37587ed71dcc? I see a version number handling regex changed there), we started seeing test failures like this in the community build:

```
[jackson-module-scala] [info]   com.fasterxml.jackson.databind.JsonMappingException: Scala module 0.0.0 requires Jackson Databind version >= 0.0.0 and < 0.1.0
```

(from https://scala-ci.typesafe.com/job/scala-2.12.x-integrate-community-build/1316/consoleFull)

if you're not able to reproduce this yourselves, let me know.

as a workaround for the time being, in the community build we have frozen the project at abacab34f065abda49c4575a8dfb941498c02b3b

**Comment by pijusn** [comment]
It fails in Travis CI ( https://travis-ci.org/FasterXML/jackson-module-scala/jobs/259139204 ) as well as locally (`sbt test`).

**Comment by nbauernfeind** [comment]
It looks like this was fixed in 5b14bd050640214d4a4f2e84cce99d8c6659df74. Please re-open if you still see this issue.

**Comment by SethTisue** [comment]
all good: https://github.com/scala/community-builds/pull/730 👍
